documentation/milestones/individual_milestones/static_site.md

```diff
@@ -0,0 +1,199 @@
+
+- [x] **Internationalization:** While there will be content delivered within this milestone that needs to be translated in the future, we do not expect to have a framework for managing translations set up by the time this milestone is delivered.
+- [x] **CMS:** While in the long-term we may want to support a Content Management Service (CMS) that allows non-technical users to update and manage copy for the website, we do not expect a CMS to be selected and implemented when we launch this site.
+
+### Open questions
+<!-- Optional -->
+
+- [x] None
+
+### Not doing
+<!-- Optional -->
+
+*The following work will* not *be completed as part of this milestone:*
+
+1. **Translating site contents:** Translation of key documents will be covered in an upcoming milestone
+2. **Legacy web analytics:** Updating the existing analytics recorded on legacy grants.gov in order to establish a baseline for comparing the site traffic for `beta.grants.gov` will happen in a later milestone
+
+## Integrations
+
+### Translations
+<!-- Required -->
+
+*Does this milestone involve delivering any content that needs translation?*
+
+Yes, the site contents will need to be translated.
+
+*If so, when will English-language content be locked? Then when will translation be started and completed?*
+
+Timeline and strategy for translation is still TBD.
+
+### Services going into PROD for the first time
+<!-- Required -->
+
+*This can include services going into PROD behind a feature flag that is not turned on.*
+
+- **Static Site:** This milestone represents the official launch of the static site
+- **beta.grants.gov Domain:** The static site is the first service to officially use the `beta.grants.gov` domain
+- **Stakeholder Feedback Form:** This is the first time we're collecting feedback directly from stakeholders on `beta.grants.gov`
+- **Web Analytics:** This will most likely be the first service for which we are configuring web analytics
+
+### Services being integrated in PROD for the first time
+<!-- Required -->
+
+*Are there multiple services that are being connected for the first time in PROD?*
+
+- **Static Site + Feedback Form:** The feedback form should be accessible directly from the site, preferably embedded directly on the page
+- **Static Site + Web Analytics:** All of the public pages on the static site should be configured to track web analytics
+- **Static Site + Communications Platforms:** The static site should link to the relevant communication platforms that are available at the time of launch
+
+### Data being shared publicly for the first time
+<!-- Required -->
+
+*Are there any fields being shared publicly that have never been shared in PROD before?*
+
+- No, the content of the static site in this milestone will be limited to informational about the project or about the NOFO prototypes that have already been published on legacy grants.gov. It does not include exposing any production data from the new NOFO data model.
```

Comment:
information*

Reply:
Good catch, fixed in [this commit](https://github.com/HHS/grants-equity/pull/243/commits/b1620b74ecc0d590e7fc06575fd8f47edc035bc6)